frontend/model/contracts/group.js

```diff
@@ -410,6 +410,12 @@ DefineContract({
         incomeAmount: x => typeof x === 'number' && x >= 0,
         pledgeAmount: x => typeof x === 'number' && x >= 0,
         nonMonetaryAdd: string,
+        paymentMethods: objectMaybeOf({
+          bitcoin: objectOf({ value: string }),
+          paypal: objectOf({ value: string }),
+          venmo: objectOf({ value: string }),
+          other: objectOf({ value: string })
+        }),
```

Comment:
Here we want `objectMaybeOf` for each of the payment methods as well, since we don't know what sort of settings the payment methods will take